src/views/errors/page-not-found.njk

```diff
@@ -1,4 +1,16 @@
 {% extends "base-page.njk" %}
+{% from "components/onPageLoad/macro.njk" import ga4OnPageLoad %}
 
 {% set hmpoPageKey = "pageNotFound" %}
 {% set gtmJourney = "kbv-hmrc - not found" %}
+
+{{ ga4OnPageLoad({
+  nonce: cspNonce,
+  statusCode: '404',
+  englishPageTitle: 'error.pageNotFound.title' | translate,
+  taxonomyLevel1: 'web cri',
+  taxonomyLevel2: 'hmrc-kbv',
+  contentId: 'undefined',
+  loggedInStatus: true,
+  dynamic: false
+}) }}
```

Comment:
Could you check the error pages - I think this may need to be within
```
{% block mainContent %}
  {{ super() }}
  ...
{% endblock %}
```
for it to include it in the page?